bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -145,6 +146,21 @@ They differ for each vehicle type, build-in sensors and activated services.
 | [tires](#tire-pressure)          | Current and wanted pressure for all tires         |  X   |  X   |    X    |  X  |
 | [image](#image)                  | Provides an image of your vehicle                 |  X   |  X   |    X    |  X  |
 
+#### Vehicle Update
+
+Provides switches to force partly updates of the BMW API functionality
```

Comment:
Maybe extend this a little to explain that the BMW API has some quota and for specific use-cases it might be needed to manually trigger an update?

Reply:
done